app/controllers/mixins/actions/vm_actions/resize.rb

```diff
@@ -9,6 +9,7 @@ def resizevms
           when "miq_request_edit"
             assert_privileges("miq_request_edit")
           else
+            # TODO: should we handle this through assert_privileges?
             raise MiqException::RbacPrivilegeException, _("The user is not authorized for this task or item.")
```

Comment:
This one is just different.  We're not asserting a privilege, just failing regardless if you got to an unsupported resize vm action button.  I wasn't sure how to deal with this so left it as is.